firmware_configurations/klipper/clee/skr-e3-mini-v3.cfg

```diff
@@ -0,0 +1,306 @@
+# This file contains common pin mappings for the BIGTREETECH SKR
+# MINI 3.0. To use this config, the firmware should be compiled for 
+# the STM32G0B1 with a "8KiB bootloader" and a "Clock Reference" set
+# to "8MHz crystal", with "Communication interface" set to "USB (on
+# PA11/PA12)"
+
+# The "make flash" command does not work on the SKR mini. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" on an SD card and then restart the SKR
+# mini with that SD card.
+
+## Voron Design VORON 0.1 SKR Mini E3 V3 config
+
+## *** THINGS TO CHANGE/CHECK: ***
+## MCU path                              [mcu] section
+## Thermistor types                      [extruder] and [heater_bed] sections - See 'sensor types' list at end of file
+## Z and Extruder motor currents         [tmc2209 stepper_*] sections. Uncomment the stepper motor you have
+## Full steps per rotation for Extruder  [extruder] section
+## Thermistor types                      [extruder] and [heater_bed] sections - See 'sensor types' list at end of file
+## Extruder motor currents               [extruder] section
+## PID tune                              [extruder] and [heater_bed] sections
+## Fine tune E steps                     [extruder] section
+## For more info                         check https://docs.vorondesign.com/build/startup/#v0
+
+[mcu]
+#####################################################################
+# Obtain definition by "ls -l /dev/serial/by-id/"
+#####################################################################
+serial: /dev/serial/by-id/usb-Klipper_stm32f103xe_REPLACE_WITH_YOUR_VALUE
```

Comment:
v3 uses an stm32g doesn't it? may want to consider removing the `usb-Klipper_stm32f103xe_` so it doesn't confuse people. something like `serial: /dev/serial/by-id/REPLACE_WITH_YOUR_VALUE`

Reply:
How about `/dev/serial/by-id/usb-Klipper_stm32g0b1xx_REPLACE_WITH_YOUR_VALUE` instead, since that's the prefix my V3 uses? Does that work? Would keep things more aligned with the upstream V2 config.

Reply:
that should be fine.